lib/core.js

```diff
@@ -503,6 +523,180 @@ function isCrossOriginPullRequest(pullRequest) {
   return head.repo.fullName !== base.repo.fullName;
 }
 
+function getOrgName(context, pullRequest) {
+
+  try {
+
+    const { head } = pullRequest;
+    const { repo } = head;
+    const { owner } = repo;
+
+    const { type, login } = owner;
+
+    if (type === 'Organization') {
```

Comment:
Why not for users, too?

In that case I guess we'd need to reject per default?

Reply:
In my understanding, the users are not able to have teams in Github, therefor they won't be able to benefit from the reviewTeams functionality of merge-me for their personal repositories.

In that case, this method returns null, which causes `getTeamReviewContext` to return null, which causes team-review mode to switch off.

Reply:
OK, so our behavior is to simply ignore team reviews in this case. 

Makes sense, I guess.